tools/topology/topology2/sof-hda-generic.conf

```diff
@@ -11,6 +11,7 @@
 <passthrough-playback.conf>
 <passthrough-capture.conf>
 <host-copier-gain-mixin-playback.conf>
+<host-copier-gain-src-mixin-playback.conf>
```

Comment:
Could these files be included instead per HDA_CONFIG e.g. in this case in file cavs-src-mixin-mixout-hda.conf? Now adding more processing variants clutters the base topology.

Reply:
@ranj063 @juimonen @jsarha fyi - feature already supported ? or todo ?